optaweb-vehicle-routing-docs/src/main/asciidoc/development-guide.adoc

```diff
@@ -69,91 +69,50 @@ To run the back end in development mode, enter the following command:
 
 [source,shell]
 ----
-mvn spring-boot:run
+mvn compile quarkus:dev
 ----
 
-=== Automatic restart
+=== Quarkus development mode
 
-https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#using-boot-devtools-restart[Automatic restart]
-is provided by Spring Boot DevTools and only works when the back end is running using <<run-spring-boot-maven-plugin, Spring Boot Maven Plugin>>.
-It scans files on the classpath, so you only need to recompile your changes to trigger application restart.
-No IDE configuration is needed.
+In development mode, the backend automatically restarts whenever you refresh the browser tab where the front end runs if there are any changes in the back-end source code or configuration.
```

Comment:
change where to and:
and the front end runs...

Reply:
The suggested change would change the meaning of the sentence. The meaning is:

The back end restarts whenever 1) you refresh the browser tab where the front end runs [and] 2) if there are any changes...

Reply:
Feel free to propose another change that would convey the intended meaning in a more simple and clearer way.